packages/kbn-generate-console-definitions/src/generate_console_definitions.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import fs from 'fs';
+import Path, { join } from 'path';
+import { ToolingLog } from '@kbn/tooling-log';
+
```

Comment:
This file is mostly WIP and probably will change a lot in follow up PRs. But still feel free to review as usual